components/cam/bld/configure

```diff
@@ -2638,6 +2638,57 @@ sub write_filepath
 	print $fh "$camsrcdir/cam/src/physics/carma/base\n";
     }
 
+    # Superparameterization; this code section needs to appear before those
+    # adding the physics and chemistry packages to override versions of files
+    # that appear in both places
+    if ($crm) {
+
+        # interface and base model code
+        print $fh "$camsrcdir/cam/src/physics/crm\n";
+        print $fh "$camsrcdir/cam/src/physics/crm/sam\n";
+
+        # microphysics
+        if ($SPCAM_microp_scheme eq 'sam1mom') {
+            print $fh "$camsrcdir/cam/src/physics/crm/sam/MICRO_SAM1MOM\n";
+        } elsif ($SPCAM_microp_scheme eq 'm2005') {
+            if(defined $opts{'use_crm_cldfrac'}) {
+                print $fh "$camsrcdir/cam/src/physics/crm/sam/MICRO_M2005FRAC\n";
+            } else {
+                print $fh "$camsrcdir/cam/src/physics/crm/sam/MICRO_M2005\n";
+            }
+        }
+
+        # turbulence closure
+        if ( defined $opts{'clubb_crm'}) {
+            print $fh "$camsrcdir/cam/src/physics/CLUBB_core\n";
```

Comment:
"physics/CLUBB_core" does not appear to be a valid path. Should this just be "physics/crm/CLUBB"?

Also, didn't we decide to remove CLUBB from the CRM? Maybe we should revisit this at the next ECP meeting.

Reply:
+1